src/css/_BRsearch.scss

```diff
@@ -345,6 +355,58 @@
     left: 50%;
     transform: translate(-50%, -50%);
   }
+
+  .BRsearch-navigation {
+    position: relative;
+    z-index: 1;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px;
+    color: $white;
+    background-color: $brColorDarkGreyBg;
+    h4 {
+      padding: 0;
+      margin: 0;
+      font-size: 15px;
+      font-weight: normal;
+      span {
+        vertical-align: middle;
+      }
+    }
+    .pagination {
+      position: absolute;
+      left: 50%;
+      top: 50%;
+      transform: translate(-50%, -50%);
```

Comment:
QUESTION: Teach me: Why use these positioners to do the centering vs. flex box?

Reply:
The element won't center if you simply rely on flexbox. There are three elements in the parent. The first is wider than the last, and the space between them and the middle one will be equal. As a result, the middle element is not centered. Attached are two screenshots illustrating first the flexbox space-between centering and the positioning present in this PR.

![Screen Shot 2020-09-28 at 6 05 36 AM](https://user-images.githubusercontent.com/39553/94419830-45cd6700-0151-11eb-90b3-513190e1061e.png)
![Screen Shot 2020-09-28 at 6 04 33 AM](https://user-images.githubusercontent.com/39553/94419844-4960ee00-0151-11eb-980d-d9b838a50803.png)